src/components/elements/maps/SingleGeolocationMap.tsx

```diff
@@ -0,0 +1,35 @@
+import { Marker } from 'react-leaflet';
+import BaseMapContainer from './BaseMapContainer';
+import { LatLon } from '@/types/geolocationTypes';
+
+interface BaseMapProps {
+  coordinates: LatLon;
+  height?: number;
+}
+
+// Map for showing 1 pin on a static map.
+// Used for display coordinates in Geolocation Input Field.
+const SingleGeolocationMap: React.FC<BaseMapProps> = ({
+  coordinates,
+  height = 230,
+}) => {
+  return (
+    <BaseMapContainer
+      key={JSON.stringify(coordinates)}
+      height={height}
+      center={[coordinates.latitude, coordinates.longitude]}
+      zoom={16}
+      dragging={false}
+      scrollWheelZoom={false}
+      doubleClickZoom
+      touchZoom
```

Comment:
minor thing,  some of these props seem to be on both single and multi location maps. Perhaps they should be default on the BaseMapContainer

Reply:
good idea, moved some to BaseMapContainer